backend/src/passwords.cpp

```diff
@@ -0,0 +1,72 @@
+#include <iostream>
+#include "passwords.hpp"
+
+namespace passwords
+{
+    std::unordered_set<std::string> generatePasswords(const int &numPasswords, const int &maxChars)
+    {
+        int minChars = 10;
+
+        if (maxChars < minChars) {
+            std::cout << "Password must have a minimum length of 10 characters\n";
+            exit(0);
+        }
+
+        std::unordered_set<std::string> passwords;
+        std::string lowercase = "abcdefghijklmnopqrstuvwxyz";
+        std::string uppercase = "ABCDEFGHIJKLMNOPQRSTUVWXYZ";
+        std::string digits = "0123456789";
+        std::string symbols = "!#$%&'()\\*+,-./:;<=>?@[]^_`{|}~\"";
+
+        while (passwords.size() < numPasswords)
+        {
+            // randomize length between minChars and maxChars
+            int length = minChars + rand() % ((maxChars + 1) - minChars);
+            std::string password = ""; 
+            bool hasLowercase = false, hasUppercase = false, hasDigit = false, hasSymbol = false;
```

Comment:
**issue**: Don't initialize multiple values in the same line; it makes code harder to maintain/refactor.